markdown/docs/user-guide.md

```diff
@@ -764,8 +748,8 @@ The server offers an [Express middleware][express] that can be integrated into a
 
 ```javascript
 var server = new FastBootServer({
-  appFile: appFile,
-  vendorFile: vendorFile,
+  appFiles: [appFile, appFile-fastboot],
```

Comment:
ohoo yeah, I meant `appFastBootFile` to denote the additional asset for overrides